articles/quickstart/native/react-native/00-login.md

```diff
@@ -86,17 +32,22 @@ yarn add react-native-auth0
 For further reference on yarn, check [their official documentation](https://yarnpkg.com/en/docs).
 :::
 
-### Link the native module
-
-To add the functionality of the React Native Auth0 module to your project you need to link it.
+### npm
 
 ```bash
-react-native link react-native-auth0
+npm install react-native-auth0 --save
 ```
 
-::: note
-For further reference on linking libraries, check [the official documentation](https://facebook.github.io/react-native/docs/linking-libraries-ios.html).
-:::
+### Additional iOS step: Install the Module Pod
+
+CocoaPods is the package management tool for iOS that React Native framework uses to install itself into your project. For the iOS native module to work on your iOS app you must first install the library Pod. If you're familiar with older React Native SDK versions this is similar to what was called _linking a native module_. Currently, the process is simpler:
```

Comment:
> CocoaPods is the package management tool for iOS that React Native framework uses to install itself into your project.

Either add "the" before "React Native framework..." or remove the word "framework".

> For the iOS native module to work on your iOS app you must first install the library Pod.

Instead of "to work on your iOS app..." consider "to work with your iOS app".

> If you're familiar with older React Native SDK versions this is similar to what was called _linking a native module_.

Add a comma after the word "versions": If you're familiar with older React Native SDK versions, this is similar to what was called _linking a native module_.

> Currently, the process is simpler:

Consider: "The process is now simplified:"

Reply:
Good feedback!